src/components/map/mapbox/layers/ClusterLayer.tsx

```diff
@@ -236,17 +358,12 @@ const ClusterLayer: FC<ClusterLayerProps> = ({
         }
```

Comment:
catch error then log it too, if you simply ignore the exception, then log something like : <...> skipped ... blah blah